audio/src/commandhandler.py

```diff
@@ -37,10 +36,8 @@ def _handle_command(self, cmd, entity_id, message):
             return self._create_error_msg("Unsupported command")
 
     def _add_sound(self, entity_id, params):
-        path = params.get("path")
-        if path is None:
-            return self._create_error_msg("Invalid path when adding audio")
-        loops = params.get("loops", 0)
+        path = params["assets"][0]
```

Comment:
Will we ever use any assets except the first one? If not, does it need to be a list?

Reply:
I was thinking that it might be nice to keep it general, don't think it will affect performance much. (e.g. if we add something like 'play_all')

Reply:
I was thinking that it's confusing to allow configuring multiple assets if only the first is used. Since we will have to edit the opus file manually until we have an editor gui, it's worth a lot to keep it clear.

But backwards compatibility is also nice of course, so I'm fine with keeping it off you insist.

Reply:
Realized we still need multiple entity IDs to control each audio clip so to use multiple assets seem weird (at least for sound), so I'll change it to asset instead.